src/integrationtests/resources/com/aws/greengrass/integrationtests/ipc/pubsub.yaml

```diff
@@ -133,4 +133,4 @@ services:
               - 'aws.greengrass#SubscribeToTopic'
               - 'aws.greengrass#PublishToTopic'
             resources:
-              - /to*/#
```

Comment:
Will you also add some more integ tests using the new authz wildcard? e.g. for mqtt with different policies than https://github.com/aws-greengrass/aws-greengrass-nucleus/blob/main/src/integrationtests/resources/com/aws/greengrass/integrationtests/ipc/mqttproxy.yaml

Reply:
Yes, good point. I should add at least one case using each wildcard in mqtt